core/src/test/java/com/epam/esm/dao/impl/GiftDaoImplTest.java

```diff
@@ -0,0 +1,180 @@
+package com.epam.esm.dao.impl;
+
+import com.epam.esm.dao.exception.GiftNotFoundException;
```

Comment:
This comment is related to all integration tests:
1) it is better to initialize embedded database not before each test method but before each class;
when I am saying better I mean that you can decrease the time during which test are running
2) please add more assertions not only for size of list but also check the whole object

Reply:
1. i need to clear and fill my DB before each test for get valid tests
2. fixed